site/content/en/docs/tasks/manage/setup_multikueue.md

```diff
@@ -74,6 +74,16 @@ kubectl apply --server-side -f https://raw.githubusercontent.com/kubernetes-sigs
 ```
 {{% /alert %}}
 
+### Kubeflow Installation
```

Comment:
Add a link to the kubeflow installation guide, to be done in the worker clusters.